k8sclient/k8sclient.go

```diff
@@ -517,3 +527,46 @@ func GetK8sNetwork(k8sclient KubeClient, k8sArgs *types.K8sArgs, confdir string)
 
 	return delegates, nil
 }
+
+
+func getPodDefaultNetworkAnnotation(client KubeClient, k8sArgs *types.K8sArgs) (string, string, error) {
+	var err error
+
+	logging.Debugf("getPodDefaultNetworkAnnotation: %v, %v", client, k8sArgs)
+	pod, err := client.GetPod(string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_NAME))
+	if err != nil {
+		return "", "", logging.Errorf("getPodDefaultNetworkAnnotation: failed to query the pod %v in out of cluster comm: %v", string(k8sArgs.K8S_POD_NAME), err)
+	}
+
+	return pod.Annotations["k8s.v1.cni.cncf.io/default-network"], pod.ObjectMeta.Namespace, nil
```

Comment:
This could be changed to indicate that is not official (NPWG) feature, like "multus/default-network" or something like that.

@rkamudhan @dougbtv can we talk about the namespace for multus original feature?

Reply:
ideas:

```
multus.cncf.io/*
multus.io/*
multus.github.io/*
```